src/components/cc-invoice-table/cc-invoice-table.js

```diff
@@ -15,15 +15,16 @@ export const PROCESSED_STATUSES = ['PAID', 'CANCELED', 'REFUNDED'];
 
 /** @type {Invoice[]} */
 const SKELETON_INVOICES = [
-  // @formatter:off
-  { emissionDate: '2020-01-01', number: '????????????', type: 'INVOICE', status: 'PENDING', total: { currency: 'EUR', amount: 10.00 } },
-  { emissionDate: '2020-02-01', number: '????????????', type: 'INVOICE', status: 'PENDING', total: { currency: 'EUR', amount: 200.00 } },
-  { emissionDate: '2020-03-01', number: '????????????', type: 'INVOICE', status: 'PENDING', total: { currency: 'EUR', amount: 3000.00 } },
-  // @formatter:on
+  { emissionDate: '2020-01-01', number: '????????????', type: 'INVOICE', status: 'PENDING', total: { currency: 'EUR', amount: 10.00 }, downloadUrl: '', paymentUrl: '', invoiceHtml: '' },
+  { emissionDate: '2020-02-01', number: '????????????', type: 'INVOICE', status: 'PENDING', total: { currency: 'EUR', amount: 200.00 }, downloadUrl: '', paymentUrl: '', invoiceHtml: '' },
+  { emissionDate: '2020-03-01', number: '????????????', type: 'INVOICE', status: 'PENDING', total: { currency: 'EUR', amount: 3000.00 }, downloadUrl: '', paymentUrl: '', invoiceHtml: '' },
```

Comment:
**🤓 nitpick:** Not important at all but those invoices used for skeleton mode are not sorted by emission date. That's it. :smile: 

Source: related story slightly mismatching visually between prod & preview.